### ISSUE TITLE
Use semantic-release

### ISSUE THREAD

**Issue by hsablonniere** [ensuite/ensuite-present#9]
I'd like to investigate using [semantic-release](https://github.com/semantic-release/semantic-release/tree/v6.3.6).

@mojavelinux WDYT?

**Comment by mojavelinux** [comment]
Absolutely. If it works out, let's use it. I'm in full support for using semantic releases, and automating them to the full extent possible.

I do believe we should get a 1.0.0 release before we start doing semantic releases though since I consider releases below 1.0.0 to be only for prototypes.

**Comment by hsablonniere** [comment]
Works for me :smile: :+1:
